### ISSUE TITLE
TypeError: unsupported format string passed to NoneType.__format__

### ISSUE THREAD

**Issue by xanoni** [prusnak/suez#23]
Any idea how to best debug this? 

Versions:
```
lnd version 0.13.0-beta commit=v0.13.0-beta
Poetry version 1.1.7
Python 3.7.3
```

lnd.conf (relevant parts):
```
listen=localhost
rpclisten=localhost:10009
bitcoin.active=true
bitcoin.mainnet=true
bitcoin.node=bitcoind
tor.active=true
tor.socks=9050
tor.v3=true
tor.v2=false
```

Log:
```
$ lncli getinfo
{
    "version": "0.13.0-beta commit=v0.13.0-beta",
    [...]
}

$ poetry run ./suez
Traceback (most recent call last):
  File "./suez", line 5, in <module>
    suez()
  File "/home/asdf/.cache/pypoetry/virtualenvs/suez-aZJboPJE-py3.7/lib/python3.7/site-packages/click/core.py", line 1137, in __call__
    return self.main(*args, **kwargs)
  File "/home/asdf/.cache/pypoetry/virtualenvs/suez-aZJboPJE-py3.7/lib/python3.7/site-packages/click/core.py", line 1062, in main
    rv = self.invoke(ctx)
  File "/home/asdf/.cache/pypoetry/virtualenvs/suez-aZJboPJE-py3.7/lib/python3.7/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/asdf/.cache/pypoetry/virtualenvs/suez-aZJboPJE-py3.7/lib/python3.7/site-packages/click/core.py", line 763, in invoke
    return __callback(*args, **kwargs)
  File "/home/asdf/suez/suez.py", line 190, in suez
    info = info_box(ln, score)
  File "/home/asdf/suez/suez.py", line 29, in info_box
    grid.add_row("score  : ", "{:,}".format(score.get(ln.local_pubkey)))
TypeError: unsupported format string passed to NoneType.__format__
```

**Comment by prusnak** [comment]
Fixed via https://github.com/prusnak/suez/pull/22 which has been just merged

**Comment by xanoni** [comment]
> Fixed via #22 which has been just merged

Bad timing, unlucky me ;-) Thank you!
